packages/core/src/util/misc/serializableObject.ts

```diff
@@ -1,9 +1,9 @@
+/* eslint-disable sonarjs/cognitive-complexity */
```

Comment:
The commit message: 
**chore(core): add missing eslint-disable in serializableObject.ts**
... would be better described as:
**chore(core): hoist eslint-disable to whole file**